includes/Modules/Tag_Manager.php

```diff
@@ -638,4 +640,28 @@ private function register_tag() {
 		}
 	}
 
+	/**
+	 * Can the Analytics module insert it's own snippet.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param boolean $original_value Original value of useSnippet setting.
+	 *
+	 * @return boolean Filtered value.
+	 */
+	private function can_analytics_use_snippet( $original_value ) {
+		$settings = $this->get_settings()->get();
+
+		// The GAPropertyID is set by Tag Manager when it's own snippet shares the GA ID
+		// of an existing, active, Analytics tag.
+		// Returning false here disables the useSnippet setting in the Analytics setting
+		// and shows a message to the user explaining that the snippet is now controlled
+		// by Tag Manager.
+		if ( ! empty( $settings['GAPropertyID'] ) && $settings['useSnippet'] ) {
+			return false;
```

Comment:
The comment here is a little off. The `gaPropertyID` will be set whenever it is configured with a GA property in its container(s). We require that this property ID is the same for both containers (if they use AMP) and that it matches the configured property ID of Analytics, if it is already set up.
So the important thing to mention here is that this disables the Analytics snippet if there is a GA tag in the configured containers, and the GTM snippet is enabled.